keymaps/postcss-sorting.cson

```diff
@@ -1,8 +1,2 @@
-'.platform-darwin atom-text-editor':
-  'cmd-shift-s': 'postcss-sorting:run'
-
-'.platform-win32 atom-text-editor':
-  'ctrl-shift-s': 'postcss-sorting:run'
-
-'.platform-linux atom-text-editor':
+'atom-text-editor':
   'ctrl-shift-s': 'postcss-sorting:run'
```

Comment:
I understand this, in fact I'm a Mac user and I do prefer `cmd`-based keystrokes, but not when they override a default shortcut. In particular `cmd+s` is the used keystroke for saving stuff, not just in Atom but pretty much everywhere and I cannot image anyone that prefers to bind it to `postcss-sorting:run` instead.